File: src/bubble_sort.rs
pub fn bubble_sort(array: &mut Vec<i32>) {
    for _ in 1..array.len() {
        for i in 1..array.len() {
            if array[i-1] < array[i] {
                continue;
            }
            array.swap(i-1, i);
        }
    }
}

File: python_sort/bubble_sort.py
def bubble_sort(array: list[int]) -> list[int]:
    for _ in range(1, len(array)):
        for i in range(1, len(array)):
            if array[i-1] < array[i]:
                continue
            array[i], array[i-1] = array[i-1], array[i]
    
    return array
